gdsc-web-backend/Controllers/FaqsController.cs

```diff
@@ -1,35 +1,45 @@
 ﻿using System.Collections.Generic;
 using gdsc_web_backend.Models;
+using Microsoft.AspNetCore.Http;
 using Microsoft.AspNetCore.Mvc;
 
 namespace gdsc_web_backend.Controllers
 {
-    
-    [ApiController] 
+    [ApiController]
     [Route("api/[controller]")]
-    public class FaqsController: ControllerBase
+    public class FaqsController : ControllerBase
```

Comment:
These changes to FaqController shoud not be here. This task is about PagesController and Pages testing. If I merge this branch and the one with faq testing then there will be a lot of conflicts